readme/_i18n/fr_FR/welcome/1_welcome_to_joplin.md

```diff
@@ -52,7 +52,7 @@ Beaucoup plus est possible, y compris l'ajout d'exemples de code informatique, d
 
 Les notes de Joplin sont organisées en une arborescence de carnets et de sous-carnets.
 
-- Sur l'appli de **bureau**, vous pouvez créer un carnet en cliquant sur "Nouveau carnet", puis vous pouvez les faire glisser et les déposer dans d'autres carnets pour les organiser comme vous le souhaitez.
+- Sur l'appli de **bureau**, vous pouvez créer un carnet en cliquant sur "Nouveau carnet", puis vous pouvez les organiser à votre guise en les faisant glisser-déposer dans d'autres carnets ou en cliquant droit et sélectionnant "Déplacer vers un carnet".
```

Comment:
(Very minor): Joplin's context menu uses "Déplacer vers le carnet":

![screenshot: Déplacer vers le carnet highlighted in the right-click menu](https://github.com/user-attachments/assets/5c1c9ccb-1953-43af-9f01-1308aa20104e)

```suggestion
- Sur l'appli de **bureau**, vous pouvez créer un carnet en cliquant sur "Nouveau carnet", puis vous pouvez les organiser à votre guise en les faisant glisser-déposer dans d'autres carnets ou en cliquant droit et sélectionnant "Déplacer vers le carnet".
```